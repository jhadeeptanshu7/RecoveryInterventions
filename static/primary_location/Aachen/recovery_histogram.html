



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="856c433f-9458-4144-b7e1-b6f1b7706121"></div>
            
          
        
      
      
        <script type="application/json" id="137084">
          {"9c5ba24e-a11e-4fe7-9fa7-040bdce0ecdc":{"roots":{"references":[{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"137008","type":"BoxAnnotation"},{"attributes":{"dimension":1,"plot":{"id":"136981","subtype":"Figure","type":"Plot"},"ticker":{"id":"136996","type":"BasicTicker"}},"id":"136999","type":"Grid"},{"attributes":{},"id":"136996","type":"BasicTicker"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"136981","subtype":"Figure","type":"Plot"},"ticker":{"id":"136992","type":"CategoricalTicker"}},"id":"136994","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"137024","type":"BasicTickFormatter"},"plot":{"id":"136981","subtype":"Figure","type":"Plot"},"ticker":{"id":"136996","type":"BasicTicker"}},"id":"136995","type":"LinearAxis"},{"attributes":{},"id":"136992","type":"CategoricalTicker"},{"attributes":{"axis_label":"Recovery Terms","formatter":{"id":"137022","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"136981","subtype":"Figure","type":"Plot"},"ticker":{"id":"136992","type":"CategoricalTicker"}},"id":"136991","type":"CategoricalAxis"},{"attributes":{},"id":"137022","type":"CategoricalTickFormatter"},{"attributes":{},"id":"137025","type":"Selection"},{"attributes":{},"id":"137024","type":"BasicTickFormatter"},{"attributes":{},"id":"137026","type":"UnionRenderers"},{"attributes":{},"id":"137003","type":"SaveTool"},{"attributes":{"overlay":{"id":"137008","type":"BoxAnnotation"}},"id":"137002","type":"BoxZoomTool"},{"attributes":{},"id":"137001","type":"WheelZoomTool"},{"attributes":{},"id":"137000","type":"PanTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"137000","type":"PanTool"},{"id":"137001","type":"WheelZoomTool"},{"id":"137002","type":"BoxZoomTool"},{"id":"137003","type":"SaveTool"},{"id":"137004","type":"ResetTool"},{"id":"137005","type":"HelpTool"}]},"id":"137006","type":"Toolbar"},{"attributes":{},"id":"137005","type":"HelpTool"},{"attributes":{},"id":"137004","type":"ResetTool"},{"attributes":{},"id":"136989","type":"LinearScale"},{"attributes":{"callback":null,"start":0},"id":"136985","type":"DataRange1d"},{"attributes":{},"id":"136987","type":"CategoricalScale"},{"attributes":{"below":[{"id":"136991","type":"CategoricalAxis"}],"left":[{"id":"136995","type":"LinearAxis"}],"plot_height":300,"plot_width":2000,"renderers":[{"id":"136991","type":"CategoricalAxis"},{"id":"136994","type":"Grid"},{"id":"136995","type":"LinearAxis"},{"id":"136999","type":"Grid"},{"id":"137008","type":"BoxAnnotation"},{"id":"137018","type":"GlyphRenderer"}],"title":{"id":"136980","type":"Title"},"toolbar":{"id":"137006","type":"Toolbar"},"x_range":{"id":"136983","type":"FactorRange"},"x_scale":{"id":"136987","type":"CategoricalScale"},"y_range":{"id":"136985","type":"DataRange1d"},"y_scale":{"id":"136989","type":"LinearScale"}},"id":"136981","subtype":"Figure","type":"Plot"},{"attributes":{"plot":null,"text":"Recovery Term Counts"},"id":"136980","type":"Title"},{"attributes":{"callback":null,"factors":["life"],"range_padding":0.01},"id":"136983","type":"FactorRange"},{"attributes":{"data_source":{"id":"137015","type":"ColumnDataSource"},"glyph":{"id":"137016","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"137017","type":"VBar"},"selection_glyph":null,"view":{"id":"137019","type":"CDSView"}},"id":"137018","type":"GlyphRenderer"},{"attributes":{"source":{"id":"137015","type":"ColumnDataSource"}},"id":"137019","type":"CDSView"},{"attributes":{"callback":null,"data":{"top":[1],"x":["life"]},"selected":{"id":"137025","type":"Selection"},"selection_policy":{"id":"137026","type":"UnionRenderers"}},"id":"137015","type":"ColumnDataSource"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"137016","type":"VBar"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"137017","type":"VBar"}],"root_ids":["136981"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('137084').textContent;
                  var render_items = [{"docid":"9c5ba24e-a11e-4fe7-9fa7-040bdce0ecdc","roots":{"136981":"856c433f-9458-4144-b7e1-b6f1b7706121"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>