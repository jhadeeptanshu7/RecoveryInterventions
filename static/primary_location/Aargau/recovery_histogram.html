



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="00ff7f48-d972-4866-9621-55fb0f30ea0b"></div>
            
          
        
      
      
        <script type="application/json" id="323316">
          {"de5e2df0-98a8-4deb-8d0d-b2902af8292a":{"roots":{"references":[{"attributes":{},"id":"323228","type":"BasicTicker"},{"attributes":{},"id":"323224","type":"CategoricalTicker"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"323213","subtype":"Figure","type":"Plot"},"ticker":{"id":"323224","type":"CategoricalTicker"}},"id":"323226","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"323256","type":"BasicTickFormatter"},"plot":{"id":"323213","subtype":"Figure","type":"Plot"},"ticker":{"id":"323228","type":"BasicTicker"}},"id":"323227","type":"LinearAxis"},{"attributes":{},"id":"323221","type":"LinearScale"},{"attributes":{"axis_label":"Recovery Terms","formatter":{"id":"323254","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"323213","subtype":"Figure","type":"Plot"},"ticker":{"id":"323224","type":"CategoricalTicker"}},"id":"323223","type":"CategoricalAxis"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"323248","type":"VBar"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.5},"x":{"field":"x"}},"id":"323249","type":"VBar"},{"attributes":{"callback":null,"data":{"top":[1,1,1,1,1,1],"x":["na","therapy","effects","urges","territory","smart"]},"selected":{"id":"323257","type":"Selection"},"selection_policy":{"id":"323258","type":"UnionRenderers"}},"id":"323247","type":"ColumnDataSource"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"323240","type":"BoxAnnotation"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"323232","type":"PanTool"},{"id":"323233","type":"WheelZoomTool"},{"id":"323234","type":"BoxZoomTool"},{"id":"323235","type":"SaveTool"},{"id":"323236","type":"ResetTool"},{"id":"323237","type":"HelpTool"}]},"id":"323238","type":"Toolbar"},{"attributes":{"below":[{"id":"323223","type":"CategoricalAxis"}],"left":[{"id":"323227","type":"LinearAxis"}],"plot_height":300,"plot_width":2000,"renderers":[{"id":"323223","type":"CategoricalAxis"},{"id":"323226","type":"Grid"},{"id":"323227","type":"LinearAxis"},{"id":"323231","type":"Grid"},{"id":"323240","type":"BoxAnnotation"},{"id":"323250","type":"GlyphRenderer"}],"title":{"id":"323212","type":"Title"},"toolbar":{"id":"323238","type":"Toolbar"},"x_range":{"id":"323215","type":"FactorRange"},"x_scale":{"id":"323219","type":"CategoricalScale"},"y_range":{"id":"323217","type":"DataRange1d"},"y_scale":{"id":"323221","type":"LinearScale"}},"id":"323213","subtype":"Figure","type":"Plot"},{"attributes":{"plot":null,"text":"Recovery Term Counts"},"id":"323212","type":"Title"},{"attributes":{"callback":null,"factors":["na","therapy","effects","urges","territory","smart"],"range_padding":0.01},"id":"323215","type":"FactorRange"},{"attributes":{"callback":null,"start":0},"id":"323217","type":"DataRange1d"},{"attributes":{},"id":"323219","type":"CategoricalScale"},{"attributes":{},"id":"323232","type":"PanTool"},{"attributes":{"dimension":1,"plot":{"id":"323213","subtype":"Figure","type":"Plot"},"ticker":{"id":"323228","type":"BasicTicker"}},"id":"323231","type":"Grid"},{"attributes":{},"id":"323237","type":"HelpTool"},{"attributes":{},"id":"323236","type":"ResetTool"},{"attributes":{},"id":"323235","type":"SaveTool"},{"attributes":{"overlay":{"id":"323240","type":"BoxAnnotation"}},"id":"323234","type":"BoxZoomTool"},{"attributes":{},"id":"323233","type":"WheelZoomTool"},{"attributes":{},"id":"323258","type":"UnionRenderers"},{"attributes":{},"id":"323254","type":"CategoricalTickFormatter"},{"attributes":{},"id":"323257","type":"Selection"},{"attributes":{},"id":"323256","type":"BasicTickFormatter"},{"attributes":{"source":{"id":"323247","type":"ColumnDataSource"}},"id":"323251","type":"CDSView"},{"attributes":{"data_source":{"id":"323247","type":"ColumnDataSource"},"glyph":{"id":"323248","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"323249","type":"VBar"},"selection_glyph":null,"view":{"id":"323251","type":"CDSView"}},"id":"323250","type":"GlyphRenderer"}],"root_ids":["323213"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('323316').textContent;
                  var render_items = [{"docid":"de5e2df0-98a8-4deb-8d0d-b2902af8292a","roots":{"323213":"00ff7f48-d972-4866-9621-55fb0f30ea0b"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>