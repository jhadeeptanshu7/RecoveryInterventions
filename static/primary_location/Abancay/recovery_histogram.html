



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="e3e52ac6-c5d6-4a4d-984e-b20e218e7d08"></div>
            
          
        
      
      
        <script type="application/json" id="239188">
          {"fbe1cc0b-b3f6-4714-805e-147515e4c4b4":{"roots":{"references":[{"attributes":{},"id":"239109","type":"HelpTool"},{"attributes":{},"id":"239108","type":"ResetTool"},{"attributes":{},"id":"239105","type":"WheelZoomTool"},{"attributes":{},"id":"239104","type":"PanTool"},{"attributes":{},"id":"239107","type":"SaveTool"},{"attributes":{"overlay":{"id":"239112","type":"BoxAnnotation"}},"id":"239106","type":"BoxZoomTool"},{"attributes":{},"id":"239100","type":"BasicTicker"},{"attributes":{"dimension":1,"plot":{"id":"239085","subtype":"Figure","type":"Plot"},"ticker":{"id":"239100","type":"BasicTicker"}},"id":"239103","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"239128","type":"BasicTickFormatter"},"plot":{"id":"239085","subtype":"Figure","type":"Plot"},"ticker":{"id":"239100","type":"BasicTicker"}},"id":"239099","type":"LinearAxis"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"239085","subtype":"Figure","type":"Plot"},"ticker":{"id":"239096","type":"CategoricalTicker"}},"id":"239098","type":"Grid"},{"attributes":{},"id":"239093","type":"LinearScale"},{"attributes":{},"id":"239091","type":"CategoricalScale"},{"attributes":{},"id":"239096","type":"CategoricalTicker"},{"attributes":{"axis_label":"Recovery Terms","formatter":{"id":"239126","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"239085","subtype":"Figure","type":"Plot"},"ticker":{"id":"239096","type":"CategoricalTicker"}},"id":"239095","type":"CategoricalAxis"},{"attributes":{"data_source":{"id":"239119","type":"ColumnDataSource"},"glyph":{"id":"239120","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"239121","type":"VBar"},"selection_glyph":null,"view":{"id":"239123","type":"CDSView"}},"id":"239122","type":"GlyphRenderer"},{"attributes":{"callback":null,"data":{"top":[1,1,1,1,1,1,1,1,1,1,1,1,1,1],"x":["death","life","anxiety","wrists","programs","groggy","insomnia","application","program","awake","clean","anger","asleep","medication"]},"selected":{"id":"239129","type":"Selection"},"selection_policy":{"id":"239130","type":"UnionRenderers"}},"id":"239119","type":"ColumnDataSource"},{"attributes":{},"id":"239130","type":"UnionRenderers"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"239112","type":"BoxAnnotation"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"239104","type":"PanTool"},{"id":"239105","type":"WheelZoomTool"},{"id":"239106","type":"BoxZoomTool"},{"id":"239107","type":"SaveTool"},{"id":"239108","type":"ResetTool"},{"id":"239109","type":"HelpTool"}]},"id":"239110","type":"Toolbar"},{"attributes":{},"id":"239126","type":"CategoricalTickFormatter"},{"attributes":{"source":{"id":"239119","type":"ColumnDataSource"}},"id":"239123","type":"CDSView"},{"attributes":{"callback":null,"start":0},"id":"239089","type":"DataRange1d"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.2},"x":{"field":"x"}},"id":"239121","type":"VBar"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.2},"x":{"field":"x"}},"id":"239120","type":"VBar"},{"attributes":{"plot":null,"text":"Recovery Term Counts"},"id":"239084","type":"Title"},{"attributes":{"below":[{"id":"239095","type":"CategoricalAxis"}],"left":[{"id":"239099","type":"LinearAxis"}],"plot_height":300,"plot_width":2000,"renderers":[{"id":"239095","type":"CategoricalAxis"},{"id":"239098","type":"Grid"},{"id":"239099","type":"LinearAxis"},{"id":"239103","type":"Grid"},{"id":"239112","type":"BoxAnnotation"},{"id":"239122","type":"GlyphRenderer"}],"title":{"id":"239084","type":"Title"},"toolbar":{"id":"239110","type":"Toolbar"},"x_range":{"id":"239087","type":"FactorRange"},"x_scale":{"id":"239091","type":"CategoricalScale"},"y_range":{"id":"239089","type":"DataRange1d"},"y_scale":{"id":"239093","type":"LinearScale"}},"id":"239085","subtype":"Figure","type":"Plot"},{"attributes":{"callback":null,"factors":["death","life","anxiety","wrists","programs","groggy","insomnia","application","program","awake","clean","anger","asleep","medication"],"range_padding":0.01},"id":"239087","type":"FactorRange"},{"attributes":{},"id":"239129","type":"Selection"},{"attributes":{},"id":"239128","type":"BasicTickFormatter"}],"root_ids":["239085"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('239188').textContent;
                  var render_items = [{"docid":"fbe1cc0b-b3f6-4714-805e-147515e4c4b4","roots":{"239085":"e3e52ac6-c5d6-4a4d-984e-b20e218e7d08"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>