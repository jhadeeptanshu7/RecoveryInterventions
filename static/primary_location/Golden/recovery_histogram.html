



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="da2a40f9-048b-4e3e-99c6-06adcffcdabb"></div>
            
          
        
      
      
        <script type="application/json" id="209056">
          {"aa41cb09-dfb0-424e-99c2-f2a110621af1":{"roots":{"references":[{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"208980","type":"BoxAnnotation"},{"attributes":{"callback":null,"data":{"top":[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],"x":["groups","life","pain","recovery","wilderness","medical","disease","project","application","antibiotics","journey","suffering","clean","diagnosis","meeting","smart","er"]},"selected":{"id":"208997","type":"Selection"},"selection_policy":{"id":"208998","type":"UnionRenderers"}},"id":"208987","type":"ColumnDataSource"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.2},"x":{"field":"x"}},"id":"208989","type":"VBar"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.2},"x":{"field":"x"}},"id":"208988","type":"VBar"},{"attributes":{"plot":null,"text":"Recovery Term Counts"},"id":"208952","type":"Title"},{"attributes":{"callback":null,"start":0},"id":"208957","type":"DataRange1d"},{"attributes":{"callback":null,"factors":["groups","life","pain","recovery","wilderness","medical","disease","project","application","antibiotics","journey","suffering","clean","diagnosis","meeting","smart","er"],"range_padding":0.01},"id":"208955","type":"FactorRange"},{"attributes":{},"id":"208968","type":"BasicTicker"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"208996","type":"BasicTickFormatter"},"plot":{"id":"208953","subtype":"Figure","type":"Plot"},"ticker":{"id":"208968","type":"BasicTicker"}},"id":"208967","type":"LinearAxis"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"208953","subtype":"Figure","type":"Plot"},"ticker":{"id":"208964","type":"CategoricalTicker"}},"id":"208966","type":"Grid"},{"attributes":{},"id":"208964","type":"CategoricalTicker"},{"attributes":{"axis_label":"Recovery Terms","formatter":{"id":"208994","type":"CategoricalTickFormatter"},"major_label_orientation":"vertical","plot":{"id":"208953","subtype":"Figure","type":"Plot"},"ticker":{"id":"208964","type":"CategoricalTicker"}},"id":"208963","type":"CategoricalAxis"},{"attributes":{},"id":"208961","type":"LinearScale"},{"attributes":{"below":[{"id":"208963","type":"CategoricalAxis"}],"left":[{"id":"208967","type":"LinearAxis"}],"plot_height":300,"plot_width":2000,"renderers":[{"id":"208963","type":"CategoricalAxis"},{"id":"208966","type":"Grid"},{"id":"208967","type":"LinearAxis"},{"id":"208971","type":"Grid"},{"id":"208980","type":"BoxAnnotation"},{"id":"208990","type":"GlyphRenderer"}],"title":{"id":"208952","type":"Title"},"toolbar":{"id":"208978","type":"Toolbar"},"x_range":{"id":"208955","type":"FactorRange"},"x_scale":{"id":"208959","type":"CategoricalScale"},"y_range":{"id":"208957","type":"DataRange1d"},"y_scale":{"id":"208961","type":"LinearScale"}},"id":"208953","subtype":"Figure","type":"Plot"},{"attributes":{"data_source":{"id":"208987","type":"ColumnDataSource"},"glyph":{"id":"208988","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"208989","type":"VBar"},"selection_glyph":null,"view":{"id":"208991","type":"CDSView"}},"id":"208990","type":"GlyphRenderer"},{"attributes":{"source":{"id":"208987","type":"ColumnDataSource"}},"id":"208991","type":"CDSView"},{"attributes":{},"id":"208996","type":"BasicTickFormatter"},{"attributes":{},"id":"208997","type":"Selection"},{"attributes":{},"id":"208994","type":"CategoricalTickFormatter"},{"attributes":{},"id":"208998","type":"UnionRenderers"},{"attributes":{"dimension":1,"plot":{"id":"208953","subtype":"Figure","type":"Plot"},"ticker":{"id":"208968","type":"BasicTicker"}},"id":"208971","type":"Grid"},{"attributes":{},"id":"208972","type":"PanTool"},{"attributes":{},"id":"208973","type":"WheelZoomTool"},{"attributes":{"overlay":{"id":"208980","type":"BoxAnnotation"}},"id":"208974","type":"BoxZoomTool"},{"attributes":{},"id":"208975","type":"SaveTool"},{"attributes":{},"id":"208976","type":"ResetTool"},{"attributes":{},"id":"208977","type":"HelpTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"208972","type":"PanTool"},{"id":"208973","type":"WheelZoomTool"},{"id":"208974","type":"BoxZoomTool"},{"id":"208975","type":"SaveTool"},{"id":"208976","type":"ResetTool"},{"id":"208977","type":"HelpTool"}]},"id":"208978","type":"Toolbar"},{"attributes":{},"id":"208959","type":"CategoricalScale"}],"root_ids":["208953"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('209056').textContent;
                  var render_items = [{"docid":"aa41cb09-dfb0-424e-99c2-f2a110621af1","roots":{"208953":"da2a40f9-048b-4e3e-99c6-06adcffcdabb"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>